### PR TITLE
Import/06 aktuelles

### PR DESCRIPTION
**Pull request by BlueEnni**
@TomasKostadinov can be merged aswell

or do you have anything else to add @leokndlr ?

### CODE REVIEW

File: structure/06 Aktuelles/2019_03TrainingslagerMallorca/2019_03TrainingslagerMallorca.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<html>
```
Comment (TomasKostadinov) [inline structure/06 Aktuelles/2019_03TrainingslagerMallorca/2019_03TrainingslagerMallorca.html:1]: Please add body tags as seen in #13

File: structure/06 Aktuelles/2019_07MaintalUltraTrail/2019_07PressemeldungMTUM_Veitshoechheim.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<p><h2>Bericht zum 6. Maintal Ultra Marathon in Veitshöchheim / Sa. 13.07.2019</h2></p>
```
Comment (TomasKostadinov) [inline structure/06 Aktuelles/2019_07MaintalUltraTrail/2019_07PressemeldungMTUM_Veitshoechheim.html]: Html and Body Tags are missing here too.

File: structure/06 Aktuelles/2019_09LaufangebotWinter/2019_09TrainingszeitenLauftreffWinter.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<h2>Trainingszeiten Lauftreff / Walkingtreff SV Mergelstetten </h2>
+
```
Comment (TomasKostadinov) [inline structure/06 Aktuelles/2019_09LaufangebotWinter/2019_09TrainingszeitenLauftreffWinter.html:2]: Same here.

**Review by TomasKostadinov — CHANGES_REQUESTED** [review]
Please fix the markup.

**Comment by BlueEnni** [comment]
@TomasKostadinov added missing tags

File: structure/06 Aktuelles/2019_03TrainingslagerMallorca/2019_03TrainingslagerMallorca.html [hunk 4]
```diff
@@ -1,13 +1,18 @@
...
-<h5>Alle 17 Triathleten wieder gesund und munter zurück.</h5>
-<p>
-    Vom 16. bis 24. März 2019 waren die Triathleten den SV Mergelstetten wieder im Trainingslager auf Mallorca. Alle 17
-    Teilnehmer sind begeistert und gesund ins kühle Heidenheim zurückgekehrt.
-    Bei unterschiedlichen Wetterbedingungen haben wir im Durchschnitt rund 1000 km und 9000 Höhenmeter zurückgelegt.
-    Einmal sind wir auch richtig nass geworden und haben uns auf eine heiße Dusche gefreut.
-    Zusätzlich sind haben einige Teilnehmer am 17. März 2019 die Chance genutzt und sind beim Halbmarathon oder auf der
-    10 km Strecke in Palma gestartet.
-</p>
+
```
Comment (TomasKostadinov) [inline structure/06 Aktuelles/2019_03TrainingslagerMallorca/2019_03TrainingslagerMallorca.html:2]: How about a `<html>` here? 😅😉

**Review by TomasKostadinov — CHANGES_REQUESTED** [review]

**Review by TomasKostadinov — APPROVED** [review]

**Pull request merged**